content/pricing/oss-project.md

```diff
@@ -0,0 +1,47 @@
+---
+title: OSS Project Pricing
+meta_desc: Pulumi's infrastructure as code SDK and SaaS products together provide plans for all organizations, including OSI licensed  open source projects.
+type: page
+layout: pricing-oss
+
+hero:
+    title: Pulumi Open Source Program
+    description: |
+        At the core of Pulumi is an open source project and an engaged community. Pulumi exists thanks
```

Comment:
this opening didn't really tell me what is available in a clear way. anyway to shorten this, and be specific that folks can get access to a paid plan for free?

Reply:
👍